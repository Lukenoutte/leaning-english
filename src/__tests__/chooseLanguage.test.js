import { render, screen, cleanup } from "@testing-library/react";
import ChooseLanguage from "../components/chooseLanguage/chooseLanguage";
import "@testing-library/jest-dom";

afterEach(() => {
  cleanup();
});

test("check choose language", () => {
  render(<ChooseLanguage />);
  const chooseLanguage = screen.getByTestId("choose-language");
  expect(chooseLanguage).toBeInTheDocument();
});
