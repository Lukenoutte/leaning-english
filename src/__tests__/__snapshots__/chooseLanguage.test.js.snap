// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check choose language 1`] = `
<div
  class="choose-language-wrapper"
  data-testid="choose-language"
>
  <div
    class="from-container g-shadow-light"
  >
    <span>
      From: 
    </span>
    <select
      name="language"
    >
      <option
        value="en"
      >
        English
      </option>
    </select>
  </div>
  <div
    class="to-container g-shadow-light"
  >
    <span>
      To: 
    </span>
    <select
      name="languages"
    >
      <option
        value="pt-br"
      >
        Portuguese (BR)
      </option>
      <option
        value="fr"
      >
        French
      </option>
      <option
        value="es"
      >
        Spanish
      </option>
      <option
        value="ja"
      >
        Japanese
      </option>
      <option
        value="ru"
      >
        Russian
      </option>
      <option
        value="sw"
      >
        Swahili
      </option>
      <option
        value="id"
      >
        Indonesian
      </option>
      <option
        value="zh-Hans"
      >
        Chinese
      </option>
    </select>
  </div>
</div>
`;
